modules/installation-and-upgrade/pages/container-management/updating-server-containers.adoc

```diff
@@ -0,0 +1,9 @@
+= Update containers
+
+The {productname} {productnumber} Server container can be updated using the following command:
+
+----
+mgradm upgrade
```

Comment:
this did not work for me.
I had to enter

  mgradm upgrade podman

@mbussolotto is podman required?

Reply:
yes it is required